deploy/docs/Installation_with_Helm.md

```diff
@@ -36,66 +41,84 @@ These steps require that no Prometheus exists. If you already have Prometheus in
 - [How to install if you have standalone Prometheus](./standAlonePrometheus.md) 
 - [How to install our Prometheus side by side with your existing Prometheus](./SideBySidePrometheus.md)
 
+The Helm chart installation requires two parameter overrides:
+* __sumologic.accessId__ - Sumo [Access ID](https://help.sumologic.com/Manage/Security/Access-Keys).
+* __sumologic.accessKey__ - Sumo [Access key](https://help.sumologic.com/Manage/Security/Access-Keys).
+
+The following parameter is optional, but we recommend setting it.
+* __sumologic.clusterName__ - An identifier for your Kubernetes cluster.  This is the name you will see for the cluster in Sumo Logic. Default is `kubernetes`.
+
 To install the chart, first add the `sumologic` private repo:
 
 ```bash
 helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection
 ```
 
-Install the chart with release name `collection` and namespace `sumologic`
+Next you can run `helm upgrade --install` to install or upgrade our chart.  An example command with the minimum parameters is provided below for Helm2 and Helm3.
```

Comment:
is there a specific reason for defaulting to `helm upgrade` even for installation?

Reply:
Yeah, great question.  You will notice that the install and upgrade command are now identical.  It makes for a completely idempotent command.  Many of our customers actually do this anyway for that reason when they bake our chart into their automation.  This is also what the helm TF provider does.